develop_requirements.txt

```diff
@@ -0,0 +1,42 @@
+appdirs==1.4.3
+asn1crypto==1.2.0
+attrs==19.3.0
+black==19.3b0
+cachetools==3.1.1
+certifi==2019.9.11
+cffi==1.13.0
+chardet==3.0.4
+Click==7.0
+codecov==2.0.15
+coverage==4.5.4
+cryptography==2.8
+Django==2.2.6
+django-fernet-fields==0.6
+django-maintenance-mode==0.13.3
+google-api-core==1.14.3
+google-auth==1.6.3
+google-cloud-core==1.0.3
+google-cloud-firestore==1.5.0
+google-cloud-storage==1.20.0
+google-resumable-media==0.4.1
+googleapis-common-protos==1.6.0
+grpcio==1.24.1
+gunicorn==19.9.0
+idna==2.8
+protobuf==3.10.0
+pyasn1==0.4.7
+pyasn1-modules==0.2.7
+pycparser==2.19
+python-dotenv==0.10.3
+pytz==2019.3
+requests==2.22.0
+rsa==4.0
+six==1.12.0
+sqlparse==0.3.0
+toml==0.10.0
+urllib3==1.25.6
+PyHamcrest==1.9.0
+ipdb==0.12
+djangorestframework==3.10.3
+-e git+https://github.com/ripoul/georide-position.git@afe45e57dd851d051d9877807909e0131b2a46f1#egg=georidePosition&subdirectory=apps
+
```

Comment:
can you explain this line ? why we need it ?

Reply:
add in order to try to fix continuous integration but does not make sense 👎